allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
okay, but please understand one simple thing - this is not the last change to allure ever, so we can add more changes iteratively. Did you know something about agile, short sprints, ci/cd etc?